From d88a6a5242738d6b4e2a351e4e02b9cca07f9ae1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cl=C3=A9ment=20Le=20Goffic?= <clement.legoffic@foss.st.com>
Date: Wed, 30 Apr 2025 13:47:33 +0200
Subject: [PATCH] arm64: dts: st: add I2C2 SCL rising/falling time property on
 stm32mp215f-dk
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add 'i2c-scl-[rising|falling]-time-ns' properties values.

Change-Id: I06c7ff83d6b9194ef7ae2ecb239c032201ce92cf
Signed-off-by: Clément Le Goffic <clement.legoffic@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/455159
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp215f-dk.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp215f-dk.dts b/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
index 9ae18794d060..5450a2a7dd3c 100644
--- a/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
@@ -329,6 +329,8 @@ &i2c2 {
 	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&i2c2_pins_a>;
 	pinctrl-1 = <&i2c2_sleep_pins_a>;
+	i2c-scl-rising-time-ns = <34>;
+	i2c-scl-falling-time-ns = <2>;
 	status = "okay";
 
 	goodix: goodix-ts@5d {
-- 
2.50.1

